plugins/module_utils/network/junos/argspec/ntp_global/ntp_global.py

```diff
@@ -0,0 +1,127 @@
+
+
+class Ntp_globalArgs(object):  # pylint: disable=R0903
+    """The arg spec for the junos_ntp_global module
+    """
+
+    def __init__(self, **kwargs):
+        pass
+
+    argument_spec = {
+        "config": {
+            "options": {
+                "authentication_keys": {
+                    "elements": "dict",
+                    "options": {
+                        "algorithm": {
+                            "choices": ["md5", "sha1", "sha256"],
+                            "type": "str",
+                        },
+                        "id": {"type": "int"},
+                        "key": {"type": "str"},
+                    },
+                    "type": "list",
+                },
+                "boot_server": {"type": "str"},
+                "broadcast_client": {"type": "bool"},
+                "broadcasts": {
+                    "elements": "dict",
+                    "options": {
+                        "address": {"type": "str"},
+                        "key": {"type": "str"},
+                        "routing_instance_name": {"type": "str"},
+                        "ttl": {"type": "int"},
+                        "version": {"type": "int"},
+                    },
+                    "type": "list",
+                },
+                "interval_range": {"type": "int"},
+                "multicast_client": {"type": "str"},
+                "peers": {
+                    "elements": "dict",
+                    "options": {
+                        "key": {"type": "int"},
+                        "peer": {"type": "str"},
+                        "prefer": {"type": "bool"},
+                        "version": {"type": "int"},
+                    },
+                    "type": "list",
+                },
+                "servers": {
+                    "elements": "dict",
+                    "options": {
+                        "key": {"type": "int"},
+                        "prefer": {"type": "bool"},
+                        "routing_instance": {"type": "str"},
+                        "server": {"type": "str"},
+                        "version": {"type": "int"},
+                    },
+                    "type": "list",
+                },
+                "source_addresses": {
+                    "elements": "dict",
+                    "options": {
+                        "routing_instance": {"type": "str"},
+                        "source_address": {"type": "str"},
+                    },
+                    "type": "list",
+                },
+                "threshold": {
+                    "options": {
+                        "action": {
+                            "choices": ["accept", "reject"],
+                            "type": "str",
+                        },
+                        "value": {"type": "int"},
+                    },
+                    "type": "dict",
+                },
+                "trusted_keys": {"elements": "str", "type": "list"},
```

Comment:
For compatibility with IOS argspec, this key has been made a list of dictionaries as follows in other platforms:
```
"trusted_keys": {
                    "type": "list",
                    "elements": "dict",
                    "no_log": False,
                    "options": {"key_id": {"type": "int"}},
                },
```